### ISSUE TITLE
Unable to mount NFS ISO store or install supplemental pack

### ISSUE THREAD

**Issue by snickson** [xcp-ng/xcp#20]
I've done a fresh install of XCP-ng on a HP DL 180 G7, all seems to work and I can fire up the XOA image.  When I tried to add an NFS ISO storage, I got errors and in the /var/log/kern.log I am seeing :
`Apr  4 21:26:44 nxnproc1 kernel: [ 3652.301148] hpsa 0000:05:00.0:             added scsi 0:0:0:0: RAID              HP       P410i            controller SSDSmartPathCap- En- Exp=1 qd=1024
Apr  4 21:26:44 nxnproc1 kernel: [ 3652.301506] hpsa 0000:05:00.0: scsi 0:0:0:0 addition failed, device not added.
`
This errors shows every time I try to add the NFS mount, but also happens when I try to install the Container supplemental pack.

**Comment by olivierlambert** [comment]
You mean iSCSI mount?

**Comment by snickson** [comment]
No it was an NFS mount, also happens when the supplement pack upload tries to save the iso. I'm going to put xenserver 7.2 on tonight to see if I can reproduce the issue, it may be a problem with the server configuration. Will update here once I've given that a go.

**Comment by snickson** [comment]
I think my problem is actually the same as this https://github.com/vatesfr/xen-orchestra/issues/1845 I'm using a qnap to serve the NFS mount. I'll investigate that further.

**Comment by olivierlambert** [comment]
Okay, keep us posted. Feel free to use `xe sr-probe` in parallel to see if you have the same problem

**Comment by snickson** [comment]
I had to mount this using the CLI as I couldn't get it to work through XOA.  Used 
`xe-mount-iso-sr 192.168.0.15:/ISO` and it worked first time.

**Comment by olivierlambert** [comment]
Oh, so it's an ISO mount, not a SR mount, right?

**Comment by snickson** [comment]
Yes, NFS ISO, I'll post some output on the other ticket tomorrow, it looks like sr-probe returns an empty XML document, might be qnap specific so more relevent to that ticket.

**Comment by olivierlambert** [comment]
Okay keep us posted with results of `xe` CLI :+1:
